src/app/child-dev-project/attendance/activity-participants-section/activity-participants-section.component.spec.ts

```diff
@@ -11,9 +13,10 @@ describe("ActivityParticipantsSection", () => {
   beforeEach(
     waitForAsync(() => {
       TestBed.configureTestingModule({
-        imports: [AttendanceModule],
+        imports: [AttendanceModule, AppButtonsModule],
```

Comment:
I think the `AppButtonsModule` is imported by the `AttendanceModule` and therefore not needed additionally here